resources/analysisTools/copyNumberEstimationWorkflow/beagle_create_fake_samples.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python
+
+import argparse
+
+def is_comment_line(line):
+    return line.startswith("#")
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--in_file', help='Input .vcf file')
+parser.add_argument('--out_file', help='Out .vcf file')
+args = parser.parse_args()
+
+vcf_infile = open( args.in_file, "r" )
+outfile    = open( args.out_file, "w" )
+
+for vcf_line in vcf_infile:
+    
+    if is_comment_line(vcf_line):
+        if vcf_line.startswith("#CHROM"):
+            vcf_line = vcf_line.rstrip().split("\t")
+            vcf_line.append('sample0')
```

Comment:
Is the "fake" sample name nothing that should be configurable?

Reply:
This should be fine, just adds a fake column for the beagle.